src/utils/links.test.ts

```diff
@@ -0,0 +1,118 @@
+import { partialMockUser } from '../__mocks__/partial-mocks';
```

Comment:
Almost unnecessary to test this file, but fine 😅

Reply:
i know right....  trying to make the tests a little less entangled... one test case at a time